index.bs

```diff
@@ -137,40 +146,49 @@ The following table provides a correspondence between IDL attribute names and co
 
 <table>
   <tr><th>IDL Attribute</th><th>Reflected ReflectARIA Content Attribute</th></tr>
+  <tr><td><dfn>reflectRole</dfn></td><td><pref>reflectrole</pref></td></tr>
+  <tr><td><dfn>reflectAriaActiveDescendantElement</dfn></td><td><pref>reflectariaactivedescendantelement</pref></td></tr>
   <tr><td><dfn>reflectAriaAtomic</dfn></td><td><pref>reflectariaatomic</pref></td></tr>
   <tr><td><dfn>reflectAriaAutoComplete</dfn></td><td><pref>reflectariaautocomplete</pref></td></tr>
-  <tr><td><dfn>reflectAriaBusy</dfn></td><td><sref>reflectariabusy</sref></td></tr>
-  <tr><td><dfn>reflectAriaChecked</dfn></td><td><sref>reflectariachecked</sref></td></tr>
+  <tr><td><dfn>reflectAriaBusy</dfn></td><td><pref>reflectariabusy</pref></td></tr>
+  <tr><td><dfn>reflectAriaChecked</dfn></td><td><pref>reflectariachecked</pref></td></tr>
   <tr><td><dfn>reflectAriaColCount</dfn></td><td><pref>reflectariacolcount</pref></td></tr>
   <tr><td><dfn>reflectAriaColIndex</dfn></td><td><pref>reflectariacolindex</pref></td></tr>
   <tr><td><dfn>reflectAriaColIndexText</dfn></td><td><pref>reflectariacolindextext</pref></td></tr>
   <tr><td><dfn>reflectAriaColSpan</dfn></td><td><pref>reflectariacolspan</pref></td></tr>
-  <tr><td><dfn>reflectAriaCurrent</dfn></td><td><sref>reflectariacurrent</sref></td></tr>
+  <tr><td><dfn>reflectAriaControlsElements</dfn></td><td><pref>reflectariacontrolselements</pref></td></tr>
```

Comment:
Now that I'm re-reading this, I'm not sure if it should be `reflectariacontrols` rather than `reflectariacontrolselements`.